src/Vlingo.Wire.Tests/Fdx/Bidirectional/SocketRequestResponseChannelTest.cs

```diff
@@ -155,11 +157,13 @@ public async Task TestThatRequestResponsePoolLimitsNotExceeded()
             _clientConsumer.UntilConsume = TestUntil.Happenings(total);
     
             for (int idx = 0; idx < total; ++idx) {
-                await Request(request + idx.ToString("D3"));
+                Request(request + idx.ToString("D3"));
+                Thread.Sleep(10);
             }
     
             while (_clientConsumer.UntilConsume.Remaining > 0) {
-                await _client.ProbeChannel();
+                _client.ProbeChannel();
+                Thread.Sleep(30);
```

Comment:
I had to add these in several places as now it's async fire and forget so the tests finishes before it get a chance to handle the message :)